platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -72,7 +72,8 @@ RUN apt-get install -y net-tools \
                        {%- if ENABLE_ASAN == "y" %}
                        libasan5 \
                        {%- endif %}
-                       libsystemd0
+                       libsystemd0 \
+                       dbus
```

Comment:
@xumia just one quick question, where to specify the version of dbus for this branch?

Reply:
@StormLiangMS , you mean for reproducible build right?
We can specify version in release branches, such as 202012, 202205, not in master branch.